draftquality/feature_lists/enwiki.py

```diff
@@ -1,6 +1,48 @@
 from revscoring.features import wikitext
 from revscoring.features.modifiers import max, sub
 from revscoring.languages import english
+from revscoring import Feature
+from revscoring.datasources import Datasource
+from nltk.corpus import sentiwordnet as swn
```

Comment:
Please add a note in the README, letting the user know to download the sentiwordnet nltk package...